postprocessing/visualization/receiver/src/Waveform.py

```diff
@@ -72,6 +72,10 @@ def normalize(self):
         self.waveforms[name] = wf / self.norm[name]
 
   def differentiate(self):
+    for name, wf in self.waveforms.items():
+      self.waveforms[name] = numpy.gradient(wf, self.time)
```

Comment:
Quick question: is the `self.time` here correct? (before, it was `dt` as below—the `self.time` would make more sense to me; but I'm asking simply to make sure)